mysql-test/suite/rocksdb/r/iterator_bounds.result

```diff
@@ -0,0 +1,13 @@
+create table t (i int primary key, j int, key(j) comment 'rev:bf5_2') engine=rocksdb;
+select *, hex(index_number) from information_schema.rocksdb_ddl where table_name = 't';
+TABLE_SCHEMA	TABLE_NAME	PARTITION_NAME	INDEX_NAME	COLUMN_FAMILY	INDEX_NUMBER	INDEX_TYPE	KV_FORMAT_VERSION	TTL_DURATION	INDEX_FLAGS	CF	AUTO_INCREMENT	hex(index_number)
```

Comment:
I'm not sure if it is safe to have INDEX_NUMBER in the .result file?  (I would expect the numbers to be different on every test re-run.  An experiment shows this is not happening, but is this something we can depend on, or just luck?)

I tried running `rocksdb-mtr --repeat=2 rocksdb.iterator_bounds` and the second run failed:
https://gist.github.com/spetrunia/54fcab13b5de5a8bf5e2218c0da808a7

The problem is that if this is not safe, I'm not aware of any solution - there's no way to cause MTR to run a test on a clean data dir (we have discussed this in MariaDB on some other occasion)

Reply:
It is possible, but hack-ish, to have a server instance start on forced clean data-dir by shutting down the current server instance, making the mtr test perform a manual bootstrap on a new temporary data-dir, then launch/restart the server instance against that temporary data-dir, perform whatever test is necessary, shutting down the server again and restarting it with defaults, then removing the temporary data-dir.  Otherwise, if it is not worth it here for the purpose of this test, probably just masking the one column out of the result file is the next best thing.

Reply:
... Maybe, just make a script that does `TRUNCATE TABLE t1` until `SELECT RIGHT(HEX(index_number), 2) FROM I_S...` returns 0xFF  ? 

This will be simpler, I think